pkgs/applications/misc/ilspy/default.nix

```diff
@@ -0,0 +1,99 @@
+{ lib
+, stdenv
+, pkgs
+
+, fetchzip
+, unzip
+, autoPatchelfHook
+, makeWrapper
+, makeDesktopItem
+
+#, stdenv.cc.cc.lib
+, lttng-ust
+, libkrb5
+, zlib
+, fontconfig
+
+, openssl_1_1
+, xorg
+, libX11
+, libICE
+, libSM
+, icu
+}:
+stdenv.mkDerivation rec {
+  pname = "ILSpy";
+  version = "7.2-rc";
+  description = ".NET assembly browser and decompiler.";
+
+  owner = "icsharpcode";
+  repo = "AvaloniaILSpy";
+
+  src = fetchzip {
+    url = "https://github.com/${owner}/${repo}/releases/download/v${version}/Linux.x64.Release.zip";
```

Comment:
```suggestion
    url = "https://github.com/icsharpcode/${pname}/releases/download/v${version}/Linux.x64.Release.zip";
```

Reply:
Why the preference of reduced readability here? This breaks the homepage meta.

Reply:
https://github.com/NixOS/nixpkgs/pull/182196#discussion_r925497550

Reply:
Not sure why the preference though? This pattern is from different existing packages, picked for the added readability.

Reply:
> This pattern is from different existing packages

Then the package uses an older style which is no longer prefered.
Substituting at all places possible makes the code way harder to read. If you only substitute what is really necessary (eg version) it makes things easier.